db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_01_20_175426) do
+ActiveRecord::Schema.define(version: 2021_01_21_161138) do
```

Comment:
It looks like dangerbot has flagged this PR for "Modified files in db/ and app/ inside the same PR!Modified files in db/ and app/ inside the same PR!"

Reply:
DB migrations have been removed and will be restored in a separate PR after this lands